packages/react-composites/src/composites/CallComposite/components/LocalDeviceSettings.tsx

```diff
@@ -116,16 +115,12 @@ export const LocalDeviceSettings = (props: LocalDeviceSettingsType): JSX.Element
   const cameraPermissionGranted = props.cameraPermissionGranted;
   const micPermissionGranted = props.microphonePermissionGranted;
 
-  let roleCanUseCamera = true;
-  let roleCanUseMic = true;
-
-  roleCanUseCamera = role === 'Consumer' ? false : true;
-  roleCanUseMic = role === 'Consumer' ? false : true;
+  const roleCanUseCamera = role === 'Consumer' ? false : true;
```

Comment:
this could probably just be 
```suggestion
  const roleCanUseCamera = role !== 'Consumer'
```

Reply:
Oh yep absolutely! Updated